tests/fixtures/identity.sql

```diff
@@ -259,6 +259,13 @@ SELECT a FROM test TABLESAMPLE(100)
 SELECT a FROM test TABLESAMPLE(100 ROWS)
 SELECT a FROM test TABLESAMPLE BERNOULLI (50)
 SELECT a FROM test TABLESAMPLE SYSTEM (75)
+SELECT a FROM test PIVOT(SUM(x) FOR y IN ('z', 'q'))
```

Comment:
it doesn't seem like your changes support

select * from test x PIVOT(...) y

Reply:
This is not legal in Snowflake, Bigquery, or SQL Server.

Reply:
ok sounsd good then your changes should be good, just the minor comments